src/config.ts

```diff
@@ -1376,6 +1382,47 @@ export const EmbedChainInfos: ChainInfoWithExplorer[] = [
 		features: ['stargate', 'ibc-transfer', 'no-legacy-stdTx', 'ibc-go'],
 		explorerUrlToTx: 'http://blockexplorer-1.testnet.saage.io/transactions/{txHash}',
 	},
+
+	{
+		rpc: 'https://tx-endpoint-test.imversed.com:443',
+		rest: 'https://query-endpoint-test.imversed.com:443',
+		chainId: 'imversed-test-1',
+		chainName: 'imversedtestnet',
+		stakeCurrency: {
+			coinDenom: 'IMV',
+			coinMinimalDenom: 'nimv',
+			coinDecimals: 6,
+			coinImageUrl: window.location.origin + '/public/assets/tokens/imv.png',
+			// coinGeckoId: 'imversedtest',
```

Comment:
Can define the coinGeckoId to be 'pool:nimv' ? I see it defined as the alternative coin id
`coinGeckoId: 'pool:nimv'`